tests/functional/test_add.py

```diff
@@ -78,7 +78,7 @@ def test_single_file(tmpdir):
     b = api.get(TEST_CONTEXT, 'test_single_file')
 
     # Assert the bundles contain the same data
-    bundle_hash, file_hash = get_hash(b.data[0]), get_hash(test_csv_path)
+    bundle_hash, file_hash = get_hash(b.data), get_hash(test_csv_path)
```

Comment:
this is exactly the API inconsistency I am trying to fix.  because you made the bundle with api.add, you had to use b.data[0]

however, if you had made the exact same bundle as the output of a pipe task, then b.data would have been sufficient.

one should not need to know how it was created to unpack the same data.

but, i imagine this is a breaking change that needs to be updated throughout the iep code.  it is a quick fix, and necessary.

Reply:
sgtm